val.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-valgrind --leak-resolution=high --track-fds=yes --leak-check=full --track-origins=yes --show-reachable=yes --suppressions=val.supp ${*}
+/usr/bin/valgrind --leak-resolution=high --track-fds=yes --leak-check=full --track-origins=yes --show-reachable=yes --suppressions=val.supp ${*}
```

Comment:
Why was this changed? I think we should leave this without a complete path. Valgrind should be in a user's path and if it isn't they can change the command accordingly.

Reply:
You're right. I changed it for some specific tests and forgot to revert it.